src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx

```diff
@@ -152,7 +152,10 @@ export default class ResultTable extends React.Component {
     }
     const context = this.canvas.getContext('2d');
     context.font = isHeader ? 'bold 14px Rubik' : 'normal 14px Rubik';
-    return Math.ceil(context.measureText(text).width);
+
+    // The measureText function sometimes slightly miscalculates text width.
```

Comment:
Does it miscalulate by just one pixel?

Reply:
I'm not sure, but 1px solves the problem for the cells this was noticably broken on (aka all the timestamp fields)